airbyte_cdk/sources/declarative/parsers/model_to_component_factory.py

```diff
@@ -2382,7 +2382,6 @@ def create_dynamic_schema_loader(
             primary_key=None,
             stream_slicer=combined_slicers,
             transformations=[],
-            use_cache=True,
```

Comment:
Why did we remove that? It seems to make sense that since the schema doesn't change over the time of the execution, we should use the cache here. Although, I don't know if we perform the same requests multiple times.

Reply:
For some connectors, we have a large number of schemas (e.g. service-now), and we treat them as async calls, which triggers an error with SQLite and prevents fetching the session. This change allows us to disable/enable caching when needed.

Reply:
Can you share the errors here for documentation purposes?

Reply:
Hmm... actually, I can't reproduce it now.

Rolling back the change.